server/controllers/comments.js

```diff
@@ -151,6 +151,93 @@ class CommentController {
       next(error);
     }
   }
+
+  /**
+   * Like a comment
+   * @static
+   * @param {object} req - The Request Object
+   * @param {object} res - The Response Object
+   * @param {object} next - The Next Middleware
+   * @return {json} - Returns json Object
+   * @memberof CommentController
+   * @static
+   */
+  static async likeComment(req, res, next) {
+    try {
+      const userId = req.decoded.id;
+      const { comment } = req;
+      const { id } = comment;
+
+      await comment.addLike(userId, id);
+
+      const payload = await singleComment(id);
+
+      return res.status(200).json({
+        status: 200,
+        message: 'Comment liked successfully.',
+        payload
+      });
+    } catch (error) {
+      next(error);
+    }
+  }
+
+  /**
+   * Unlike a comment
+   * @static
+   * @param {object} req - The Request Object
+   * @param {object} res - The Response Object
+   * @param {object} next - The Next Middleware
+   * @return {json} - Returns json Object
+   * @memberof CommentController
+   * @static
+   */
+  static async unlikeComment(req, res, next) {
+    try {
+      const userId = req.decoded.id;
+      const { comment } = req;
+      const { id } = comment;
+
+      await comment.removeLike(userId, id);
+
+      const payload = await singleComment(id);
+
+      return res.status(200).json({
+        status: 200,
+        message: 'Comment unliked successfully.',
+        payload
+      });
+    } catch (error) {
+      next(error);
+    }
+  }
+
+  /**
+   * Get comment like count
+   * @static
+   * @param {object} req - The Request Object
+   * @param {object} res - The Response Object
+   * @param {object} next - The Next Middleware
+   * @return {json} - Returns json Object
+   * @memberof CommentController
+   * @static
+   */
+  static async likeCount(req, res, next) {
```

Comment:
I really don't see this need for this.

Modify the `singleComment` and `comments` method to include the number of likes as an attribute in the `Comment.findOne` and `Comment.findAll` methods

Reply:
Feedback resolved.